src/Microsoft.AspNet.Server.Kestrel/Networking/PlatformApis.cs

```diff
@@ -24,12 +25,16 @@ static PlatformApis()
                 IsDarwin = string.Equals(GetUname(), "Darwin", StringComparison.Ordinal);
             }
 #endif
+
+            IsMono = PlatformServices.Default.Runtime.RuntimeType == "Mono";
```

Comment:
I think we will need/be able to get rid of this check and the entire `[DllImport("__Internal"` hack once we move away from dnx. Users will have to install libuv on Mac when using Mono.